### DISCUSSION TITLE
Charts and stats site is not visisble

### DISCUSSION THREAD

**Discussion by Neoblock8** [blockscout/blockscout#9634] [Q&A]
Hi,
 Running blockscout with docker-compose. In frontend, charts and stats page data are not shown. 
 Can i add any URL in env file?

**Comment by GimluCom** [comment]
Add your IP/domain to `NEXT_PUBLIC_API_HOST` and `NEXT_PUBLIC_APP_HOST` in [envs/common-frontend.env](https://github.com/blockscout/blockscout/blob/master/docker-compose/envs/common-frontend.env)

**Reply by Neoblock8** [reply]
Hi @GimluCom,
 Already added domain in API and APP HOST. Please find the below image.
 This is the result in console:
  https://<domain_name>/api/v1/lines 400 (Bad Request) 
{"message":"Params 'module' and 'action' are required parameters","result":null,"status":"0"}

 
![image](https://github.com/blockscout/blockscout/assets/81068101/28c6a540-ffc2-4b3d-840c-d2c28d73b483)

**Reply by GimluCom** [reply]
Please try `NEXT_PUBLIC_STATS_API_HOST=https://<domain_name>:8080`

**Reply by Neoblock8** [reply]
Still same error @GimluCom 

Stats container also running
![image](https://github.com/blockscout/blockscout/assets/81068101/2026e66b-dc01-4b78-9154-09f97811539f)

**Reply by GimluCom** [reply]
Please screenshot the error. Also I am not seeing your frontend and proxy there, but your nginx proxy should be listening to port 8080 by default.

**Reply by Neoblock8** [reply]
This is the screenshot of stats container logs. Running fine
![image](https://github.com/blockscout/blockscout/assets/81068101/f5f71a6b-999f-42b9-aa47-0ea095ae69b1)

The below screenshot is the error of hitting charts and stats button:
![image](https://github.com/blockscout/blockscout/assets/81068101/eaee98d5-5140-479d-97a6-ee524c2ddfff)

**Reply by GimluCom** [reply]
> Please try `NEXT_PUBLIC_STATS_API_HOST=https://<domain_name>:8080`

Based on the error, it does not look like you have set `:8080`. Make sure to do `docker compose down` and then `docker compose up -d` after updating ENVs.

**Reply by Neoblock8** [reply]
Tried. Still same

**Reply by GimluCom** [reply]
No, it is not the same error. You should have mentioned that you are using cloudflare as it looks like you might not have configured port 8080 on your cloudflare proxy and additionally [cloudflare does not support https on port 8080](https://developers.cloudflare.com/fundamentals/reference/network-ports/#network-ports-compatible-with-cloudflares-proxy). Please properly configure your cloudflare proxy and consider using a different port if you want to use https. Using the same port with a different subdomain is also possible.

![image](https://github.com/blockscout/blockscout/assets/79271880/64b8930e-2755-4126-9025-a32f933f5396)

**Reply by yasindce1998** [reply]
For this can we use
Nginx config like this
listen 8080 ssl;

then the ssl certificate from let's encrypt?

I am having an exact issue like this. 

If you need any information from me, like default.conf.template please tell me.

**Comment by jhkangah** [comment]
May I ask if it has been resolved? I have also encountered a similar problem
![Uploading df56cc8c-ebc1-4492-b655-f1f68f9a7bfe.png…]()

**Comment by jhkangah** [comment]
![Uploading df56cc8c-ebc1-4492-b655-f1f68f9a7bfe.png…]()
